### PR TITLE
Changelog updates - November

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1752

Adds descriptions to the changelog for work done in admin repos during November.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+  - Storing existing geometries in alt-geometry files
+
+### VARIOUS
+- Fixed incorrect concordances and name translations in a locality record in Norway ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1730))
+  - Fixed by: https://github.com/whosonfirst-data/whosonfirst-data-admin-no/pull/7
+- Updating the locality geometry of a locality in Honduras ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1736)):
+  - Fixed by: https://github.com/whosonfirst-data/whosonfirst-data-admin-hn/pull/6
+- Minor updates to locality records in Poland ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1738))
+  - Fixed by: https://github.com/whosonfirst-data/whosonfirst-data-admin-pl/pull/14
+- Update name translations in various locality records ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1743))
```
Comment (nvkelso) [inline CHANGELOG.md]: Please list out country names here

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+
+### SCOTLAND
+- Updated neighbourhood geometries in Glasgow ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1724))
+- Fixed by: https://github.com/whosonfirst-data/whosonfirst-data-admin-gb/pull/18
+  - Clipping existing geometries to the Glasgow locality geometry
+  - Flagging each updated record with a `mz:is_current` property value of `1`
+  - Storing existing geometries in alt-geometry files
+
+### VARIOUS
+- Fixed incorrect concordances and name translations in a locality record in Norway ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1730))
```
Comment (nvkelso) [inline CHANGELOG.md]: Please bold the country names when they are "loose" like this (in a Various heading).

Comment (stepps00) [reply CHANGELOG.md]: fixed in 8306db1

File: CHANGELOG.md (hunk 1 above)
Comment (stepps00) [reply CHANGELOG.md]: fixed in 8306db1

**Comment by stepps00** [comment]
Adjusted formatting, per comments - merging.

**Pull request merged**